src/System.ServiceModel.Security/pkg/System.ServiceModel.Security.pkgproj

```diff
@@ -1,6 +1,9 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
+  <PropertyGroup>
+    <PreventImplementationReference>true</PreventImplementationReference>
```

Comment:
Why did you add this? My understanding is this prevents applications from referencing the package and is meant for usage with private packages.